README-PoC.md

```diff
@@ -0,0 +1,71 @@
+# GraphQL Gateway PoC
+
+## Description
```

Comment:
Please add information how we synchronize code between 2 components: currently, we have a copy of `director.pb.go` in 2 places thanks to:
```
//go:generate protoc --go_out=plugins=grpc:. director.proto
//go:generate protoc --go_out=plugins=grpc:../../gateway/protobuf/ director.proto
```

I really like this idea but we should describe it's advantages.